gamestonk_terminal/cryptocurrency/cryptocurrency_helpers.py

```diff
@@ -238,6 +299,7 @@ def _create_closest_match_df(
     return df.merge(coins, on="id")
 
 
+# pylint: disable=R0912
 # TODO: verify vs, interval, days, depending on source
 def load(
```

Comment:
do you think changing the default to `yf` makes sense here?

Reply:
I've discussed this topic with @jose-donato. We decided not to make it the default as yf is not the best source for crypto. It has a limited number of tickers, as opposed to other data sources such as CoinPaprika.

Reply:
ok